### PR TITLE
docs: Update the documentation link in Readme file

### LINKED ISSUE #1064: Documentation link broken
**Describe the bug**
The documentation link inside the Usage section in Readme file is broken. It shows a page not found error upon visiting that link.
https://github.com/konveyor/move2kube/blob/a49a808a8840c32cabe9f31ccbd7ea8cf3c48be6/README.md?plain=1#L66

**To Reproduce**
1. Go to https://github.com/konveyor/move2kube/blob/main/README.md
2. Scroll down to Usage section
3. Click on Documentation 
4. See Page Not Found

**Expected behavior**
Open up Move2Kube Documentation

**Screenshots**
image

**Desktop (please complete the following information):**
 - OS: [e.g. RHEL]
 - Version [e.g. 22]. `move2kube version` should return it.

**Additional context**

### PR DESCRIPTION
**Pull request by Prakhar-Agarwal-byte**
I have updated the documentation link to point to the [home page ](https://move2kube.konveyor.io/). 

Fixes #1064

### CODE REVIEW

**Comment by Akash-Nayak** [comment]
@Prakhar-Agarwal-byte  Can you please update the documentation link to point to https://move2kube.konveyor.io/commands. Thanks!

**Comment by Akash-Nayak** [comment]
Also, please change the label in the PR title from `Fix:` to `fix:`, so that it passes the `label_pr` check.

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ More detailed instructions can be found in the [Move2Kube UI repo](https://githu
 
 `move2kube transform -s src`, where `src` is the folder containing the source artifacts.
 
-Checkout the [Tutorials](https://move2kube.konveyor.io/tutorials) and [Documentation](https://move2kube.konveyor.io/documentation) for more information.
```
Comment (Akash-Nayak) [inline README.md:66]: Please update the documentation link to point to https://move2kube.konveyor.io/commands.

**Review by Akash-Nayak — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Prakhar-Agarwal-byte) [reply README.md:66]: Done!

File: common/vcs/m2k-remote-temp/test-clone/konveyor/move2kube.git
Comment (Akash-Nayak) [inline common/vcs/m2k-remote-temp/test-clone/konveyor/move2kube.git:1]: Is this file required in this PR?

**Review by Akash-Nayak — CHANGES_REQUESTED** [review]

File: common/vcs/m2k-remote-temp/test-clone/konveyor/move2kube.git
Comment (Prakhar-Agarwal-byte) [reply common/vcs/m2k-remote-temp/test-clone/konveyor/move2kube.git:1]: This file is coming up automatically when I commit

Comment (Akash-Nayak) [reply common/vcs/m2k-remote-temp/test-clone/konveyor/move2kube.git:1]: Did you run `git add -A` before running `git commit`? 

Can you please try running the below commands, and see if it deletes the file.

```console
$ git rm common/vcs/m2k-remote-temp/test-clone/konveyor/move2kube.git

$ git status

$ git commit -s -m "fixup! docs: Update the documentation link in Readme file"

$ git push
```

Comment (Prakhar-Agarwal-byte) [reply common/vcs/m2k-remote-temp/test-clone/konveyor/move2kube.git:1]: Fixed it

Comment (Akash-Nayak) [reply common/vcs/m2k-remote-temp/test-clone/konveyor/move2kube.git:1]: Thanks! Can you also please rebase the branch (https://github.com/konveyor/move2kube/blob/main/contributing.md#pull-request-process), as it is out-of-date with the base branch?

Comment (Prakhar-Agarwal-byte) [reply common/vcs/m2k-remote-temp/test-clone/konveyor/move2kube.git:1]: Yup!

**Review by Akash-Nayak — APPROVED** [review]
LGTM!
